File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int[] alphabet = new int[26];

        String str = sc.nextLine();

        for (int i = 0; i < str.length(); i++) {
            alphabet[str.charAt(i) - 97]++;
        }

        for (int i : alphabet) {
            System.out.print(i + " ");
        }
    }
}

File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.cc
#include <iostream>
using namespace std;

int main() {
	
	int alphabet[26] = { 0 };

	string str;
	cin >> str;

	for (int i = 0; i < str.length(); i++)
	{
		alphabet[str[i] - 97]++;
	}

	for (int i = 0; i < 26; i++)
	{
		cout << alphabet[i] << " ";
	}
}